docs/learn/releases.rst

```diff
@@ -0,0 +1,93 @@
+Releases
+========
+
+.. note:: Some features of releases are still in beta. If you'd like to check them out, consider joining our early adopter program.
+
+
+What is a Release?
+------------------
+
+Releases are used by Sentry to provide you with additional context
+when determining the cause of an issue. You set a release context in
```

Comment:
i'll iterate on this later in the day if that's ok with you. i want to get this in so it's not blocking product changes